python/azureml/yml/env.yml

```diff
@@ -0,0 +1,23 @@
+# Conda environment specification. The dependencies defined in this file will
+# be automatically provisioned for runs with userManagedDependencies=False.
+
+# Details about the Conda environment file format:
+# https://conda.io/docs/user-guide/tasks/manage-environments.html#create-env-file-manually
+
+name: project_environment
+dependencies:
+  # The python interpreter version.
+  # Currently Azure ML only supports 3.5.2 and later.
+- python=3.6.2
+
+- pip:
+  - numpy
+  - onnxruntime
```

Comment:
This environment seems for CPU only. For GPU environment, need onnxruntime-gpu package.